.github/actions/spell-check/expect.txt

```diff
@@ -738,33 +784,33 @@ KEYBOARDMANAGEREDITOR
 KEYBOARDMANAGEREDITORLIBRARYWRAPPER
 keyboardmanagerstate
 keyboardmanagerui
+keyboardtester
 KEYEVENTF
 KEYIMAGE
 keynum
 keyremaps
 keyvault
 KILLFOCUS
 killrunner
+kinda
 kmph
-KSPROPERTY
 Kybd
 lastcodeanalysissucceeded
 Lastdevice
 LASTEXITCODE
 LAYOUTRTL
-lcb
 LCIDTo
 Lclean
 Ldone
 Ldr
-ldx
 LEFTSCROLLBAR
 LEFTTEXT
 LError
 LEVELID
 LExit
 lhwnd
 LIBID
+libraryincludes
```

Comment:
Being added because there's a comment that says libraryincludes.h 
Feels unneeded?